vendor/bat-native-ledger/src/bat/ledger/internal/bat_publishers.cc

```diff
@@ -461,10 +461,14 @@ void BatPublishers::synopsisNormalizerInternal(
     }
     if (percents.size() != 0) {
       if (totalPercents > 100) {
-        percents[valueToChange] -= 1;
+        if (percents[valueToChange] != 0) {
+          percents[valueToChange] -= 1;
+        }
         totalPercents -= 1;
```

Comment:
this one needs to go into if as well, as we didn't actually reduce total % if `percents[valueToChange]` is 0

Reply:
Good catch. Fixed.